homeassistant/components/atome/sensor.py

```diff
@@ -135,145 +134,145 @@ def _retrieve_live(self):
     @Throttle(LIVE_SCAN_INTERVAL)
     def update_live_usage(self):
         """Return current power value."""
+        _LOGGER.debug("Live Data : Update Usage")
         if not self._retrieve_live():
             _LOGGER.debug("Perform Reconnect during live request")
-            self.atome_client.login()
+            self._atome_client.login()
             self._retrieve_live()
 
-    def _retrieve_period_usage(self, period_type):
+    @property
+    def extra_state_attributes(self):
+        """Return the state attributes of this device."""
+        attr = {ATTR_ATTRIBUTION: ATTRIBUTION}
+        attr["subscribed_power"] = self._subscribed_power
+        attr["is_connected"] = self._is_connected
+        return attr
+
+    @property
+    def state(self):
+        _LOGGER.debug("Live Data : display")
+        return self._live_power
+
+    async def async_update(self):
+        """Fetch new state data for this sensor."""
+        _LOGGER.debug("Async Update sensor %s", self._name)
+        # Watch Out : only method name is given to function i.e. without ()
+        await self._hass.async_add_executor_job(self.update_live_usage)
```

Comment:
I recommend looking at using our data update coordinator for each client endpoint instead of the current approach with throttles etc.

https://developers.home-assistant.io/docs/integration_fetching_data#coordinated-single-api-poll-for-data-for-all-entities

Reply:
Thanks, please is there an easy example ?

Reply:
I look in HA components.
If there are several throttles what is the best way ?
I mean , one coordinator seems for one time rate.

Live data is at higher pace than week/mont/year.

Reply:
Probably create one coordinator per update interval and or api call group.

Reply:
Implementation of 24/11/2021 allow the use of coordinators